### DISCUSSION TITLE
Ongoing support (new K8s versions), support for AMZLinux2, bugfixes, etc.

### DISCUSSION THREAD

**Discussion by andraspatka** [nestybox/sysbox#838] [General]
Hello!
First I would like to say that I tried this out in a kubernetes cluster to solve the docker-in-docker limitations and it works just flawlessly!
I would be really keen to roll this out on a project that I'm working on, but there are a couple of concerns, as it's an enterprise project:
- Are there LTS versions? Is there a support plan for the current versions (bugfixes, security fixes, etc.)?
- Is there ongoing support for new k8s versions?
- I saw that there is planned support for AMZLinux2, is this ongoing?

I have the feeling that this project is unfortunately not used by that many people and is possibly heading to abandonware status in the near future. I find this really unfortunate as this is really an amazing piece of software which solves an issue that is pretty common with Ci/CD pipelines (maybe people just mount docker.sock and call it a day instead...).

Also if you maybe have some numbers on how many people/organizations/projects are using this technology, that would also be really helpful!

Thank you!

**Comment by ctalledo** [comment]
Hi @andraspatka, thanks for giving Sysbox a try.

The Sysbox project is sponsored by Docker Inc., it's development is ongoing, and will not be abandoned. In fact, an enhanced version of Sysbox (formerly known as Sysbox Enterprise Edition) is used in Docker Desktop (enterprise subscription only) as well as in other projects inside Docker Inc.  

Having said that, the Sysbox community edition is free and maintained and supported best-effort only, meaning that we can't commit to timelines at this time. This is because the developers that maintain Sysbox (mainly @rodnymolina  and myself) have other responsibilities inside Docker too, and we juggle those with Sysbox development & maintenance.

> Is there ongoing support for new k8s versions?

Yes, we continuously add support for newer K8s versions, but usually trail behind by a couple of versions from the latest. 

> I saw that there is planned support for AMZLinux2, is this ongoing?

I have to recheck, but I believe Sysbox already works on Amazon Linux if build from source ([it's easy](https://github.com/nestybox/sysbox/blob/master/docs/developers-guide/build.md#sysbox-developers-guide-building--installing)); we don't yet have a package installer though, and are not working on that at the moment.

Hope that helps.
